### PR TITLE
Issue 388

### PR DESCRIPTION
**Pull request by chaitanya-baraskar**
Issue - #388 

Added progress bar for better user experience.
Created separate utils package to remove interdependency and refactored code accordingly.

### CODE REVIEW

File: cmd/image.go [hunk 1]
```diff
@@ -2,10 +2,13 @@ package cmd
 
 import (
 	"fmt"
+	"github.com/briandowns/spinner"
```
Comment (eyberg) [inline cmd/image.go:5]: i'm not opposed to using a new spinner but don't we have an existing one?

File: utils/utils.go [hunk 2]
```diff
@@ -7,20 +7,27 @@ import (
...
+	ErrorColor = "\033[1;31m%s\033[0m"
+)
+
+func ExitWithError(errs string) {
 	fmt.Println(fmt.Sprintf(ErrorColor, errs))
 	os.Exit(1)
 }
 
-func arrayToString(a []int, delim string) string {
+func ArrayToString(a []int, delim string) string {
```
Comment (eyberg) [inline utils/utils.go:22]: govet will complain here

**Review by eyberg — CHANGES_REQUESTED** [review]
I don't care which one you pick but I think we are already using https://github.com/schollz/progressbar/ as a progressbar - i'd just do one or the other - no need for 2, everything else looks fine

**Comment by chaitanya-baraskar** [comment]
@eyberg this is how spinner works https://asciinema.org/a/zANzbg2cnEieFGyVxnmAg2vAY

**Comment by eyberg** [comment]
ok, well in that case maybe we should just use the existing progressbar - the whole point of that tkt was to provide an indicator of progress (25% done, 50% done, etc.) - there are facilities in the aws sdk to show progress for things like that

**Comment by chaitanya-baraskar** [comment]
Ok. I am not aware of this feature from SDK let me do some research.

**Pull request closed without merging**